src/Components/v2/AuctionCard.tsx

```diff
@@ -16,32 +16,47 @@ import { createFragmentContainer, graphql } from "react-relay"
 import { get } from "Utils/get"
 import { Media } from "Utils/Responsive"
 
-const zone = time => {
-  return moment(time, "YYYY-MM-DD").tz("America/New_York")
+const zone = timeIn => {
+  const time = moment(timeIn, "YYYY-MM-DD").tz("America/New_York")
+  const now = moment()
+  if (time.diff(now, "days") >= 1) {
+    return `${time.diff(now, "days")}d`
+  } else if (time.diff(now, "hours") >= 1) {
+    return `${time.diff(now, "hours")}h`
+  } else if (time.diff(now, "minutes") >= 1) {
+    return `${time.diff(now, "minutes")}m`
+  }
+  return `${time.diff(now, "seconds")}s`
```

Comment:
This looks like the type of time related code we’d want in MP, no?

Reply:
Yeah, I agree in principle, but there are complications. Consider the difference between a specific time, and an interval: clients that have a countdown timer, for example, wouldn't be able to use an interval specified in MP (because, as time passes, the interval changes). So I think this kind of real-time, constantly-changing value doesn't make sense to model on MP. /cc @kierangillen

Reply:
Ah gotcha 👍

What about that hardcoded NYC TZ? Why is that in there?

Reply:
I included it to match the previous behaviour, but I'll take another look to see. We don't actually _format_ the time in this function, so it can probably be removed.

Reply:
We are hardcoding many timestamps into NYC TZ currently in Force as well. This had been the status quo. I'm interested in reevaluating all of these fields when we take inventory.